### ISSUE TITLE
Mamba training loss at 7B parameters

### ISSUE THREAD

**Issue by manuel-tran** [evo-design/evo#15]
Dear authors,

Very impressive work! I have just finished reading your paper and would be very curious about the training loss curve for a 7B Mamba model as shown in Figure S4 in the paper for Transformer++, Hyena, and Striped Hyena. Do you happen to have it?

Best wishes.

**Comment by Zymrael** [comment]
We will include a comparison in the next version of the preprint

**Comment by exnx** [comment]
Mamba 7B included in [preprint v2](https://www.biorxiv.org/content/10.1101/2024.02.27.582234v2) now
